ies.md

```diff
@@ -235,13 +237,123 @@ Otherwise, things to watch out for in the UML are:
 	<li>"model" property removed - now use rdf:type</li>
 </ul>
 
+<b>v4.3.0 - Changes resulting from project engagements</b>
+1. Added <b>Stuff and Count </b>pattern, including addition of:
+<ul>
+	<li><b>Stuff</b> class</li>
+	<li><b>FiniteClassOfElement</b> class</li>
+	<li><b>finiteMembershipCount</b> attribute</li>
+	<li><b>pluriverse </b>as an instance of Element</li>
```

Comment:
Pluriverse is mentioned here, but doesn't seem to appear in the N3 file diff? Are there usage notes? Is uncapitalised because it's an instance?

Reply:
Yes but I am open to suggestions that it should be capitalised as there is only one. single pluriverse. @MonkeyChap @paulc-dstl @limartor what do you all think?

Reply:
I would argue for consistency. If only classes are capitalised, and pluriverse is not a class, then it should surely not be capitalised.